### PR TITLE
Added H1 to home page

### PR DESCRIPTION
**Pull request by Pedro-Pan**
image

Do you guys like my new title ? 
please @nnevedm tell me what you think :heart:

### CODE REVIEW

**Review by nnevedm — COMMENTED** [review]
yes i approve !

**Review by nnevedm — APPROVED** [review]

**Review by nnevedm — APPROVED** [review]
good job young padawan

**Comment by Pedro-Pan** [comment]
Its been done boss
image

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,2 +1 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<h1>We made it babyy !</h1>
```
Comment (Pedro-Pan) [inline app/views/pages/home.html.erb]: I actulaly preferd 3 exclamations marks .

**Pull request merged**